### DISCUSSION TITLE
Import OrbitControls seems to pull in all of `three-stdlib`

### DISCUSSION THREAD

**Discussion by stephent** [pmndrs/drei#1660] [Q&A]
For example, per VS Code:

<img width="723" alt="Screenshot 2023-09-23 at 17 39 21" src="https://github.com/pmndrs/drei/assets/898090/e640716d-0bac-49f9-99a9-e33a02789c41">

<img width="718" alt="Screenshot 2023-09-23 at 17 39 43" src="https://github.com/pmndrs/drei/assets/898090/72e0278e-6eda-422b-a55d-86819cf077a1">

When I build (vite project using rollup), all of three-stdlib is included in the build output.

Are there any workarounds for this, or known issues with tree-shaking? Thanks!

Versions:
```
    "@react-three/drei": "9.84.2",
    "@react-three/fiber": "8.14.2",
    "three": "0.156.0"
```

**Comment by EvenTopland** [comment]
Hi, I'm having the same issue. Did you find a solution?

**Reply by stephent** [reply]
I didn't, unfortunately. I haven't dug into it too far, but I recall reading various threads and posts on the three.js forum that [suggest](https://discourse.threejs.org/t/vite-rollup-doesnt-tree-shake-threejs-when-using-r3f/68277) the issue may lie there. I get the impression there's an assumption that any web app using three.js is likely to be "heavy" anyway, and that tree shaking is not a high priority.

**Reply by EvenTopland** [reply]
That’s unfortunate, package size really makes a difference.
I made an [issue](https://github.com/pmndrs/drei/issues/2292) here after encountering the same problem, and it seems like they solved the issue once in [three-stdlib](https://github.com/pmndrs/three-stdlib/issues/287), but the issue has since come back
